src/sql/base/browser/ui/table/formatters.ts

```diff
@@ -85,6 +85,9 @@ export function slickGridDataItemColumnValueExtractor(value: any, columnDef: any
  */
 export function slickGridDataItemColumnValueWithNoData(value: any, columnDef: any): { text: string; ariaLabel: string; } {
 	let displayValue = value[columnDef.field];
+	if (typeof displayValue === 'number') {
```

Comment:
we want to try for only number or anything not string?

Reply:
The API specifies columns to be a `string[] | number[]`, so it will probably failure before itself.